.github/workflows/test.yml

```
@@ -13,6 +13,7 @@ concurrency:
 jobs:
   build:
     name: ${{ matrix.noxenv }}
+    if: ${{ (github.repository_owner == 'pypa' && github.event_name == 'schedule') || github.event_name != 'schedule' }}
```

I think this could be shortened to:

```suggestion
    if: ${{ github.repository_owner == 'pypa' || github.event_name != 'schedule' }}
```

---

Looks good, let's try it :)